e2e/testcases/helm_sync_test.go

```diff
@@ -294,7 +293,7 @@ image:
 	if err := nt.KubeClient.Update(cm3); err != nil {
 		nt.T.Fatal(err)
 	}
-	nt.WaitForRootSyncStalledError(rs.Namespace, rs.Name, "Validation", "KNV1061: RootSyncs must reference ConfigMaps with valid spec.helm.valuesFileSources.valuesFile")
+	nt.WaitForRootSyncStalledError(rs.Namespace, rs.Name, "Validation", "KNV1061: RootSyncs must reference ConfigMaps with valid spec.helm.valuesFileRefs.valuesFile")
```

Comment:
what's a `spec.helm.valuesFileRefs.valuesFile`?

Reply:
oooh. valuesFile is supposed to be the name of the ConfigMap data key? I thought we renamed that....
Maybe that could be in a follow up PR.  It should just be `dataKey` or something.

Reply:
Don't we just default the valuesFile value anyway? So this error should just say "valid spec.helm.valuesFileRefs"

Reply:
Yeah it's the field name for the data key. I think I originally just called it `key` and there was some back and forth. I'm ok with `dataKey`, but we should get consensus from @janetkuo. 

I'll add a link to this comment thread to the umbrella issue so that we can track following up.

Reply:
yes that error message seems clearer.

Reply:
We default the valuesFile but user is allowed to specify some other valuesFile if they want to. So I think it might be nice to include the valuesFile in the error message.

Reply:
Updated. LMK what you think